hw/ip/otbn/data/otbn_sec_cm_testplan.hjson

```diff
@@ -27,43 +27,43 @@
       name: sec_cm_mem_scramble
       desc: "Verify the countermeasure(s) MEM.SCRAMBLE."
       milestone: V2S
-      tests: []
+      tests: ["otbn_imem_err", "otbn_dmem_err"]
```

Comment:
I am not sure if `Xmem_err` tests are suitable for checking `MEM.SCRAMBLE` countermeasure. But also I am not sure which type of a test would verify if we are actually scrambling the memories or not. Maybe we need a new test that just plainly tries to read/write to the memories (rather than using special functions from `sram_scrambler_pkg`)